### DISCUSSION TITLE
Add "lock BPM" function to Double time mod

### DISCUSSION THREAD

**Discussion by emilow12** [ppy/osu#28256] [Ideas] [answered]
I think the title is quite self-explanatory.
An option to lock the double time mod to certain bpm ( either of current map or writeable ), not sure how to handle if it exceeds 2x multiplier though ( maybe cap it to whatever 2x is, in such case? ) also not sure if it goes below 1x,  automatically change mod to half time then?

I think it would be really cool addition to the game and make many folks happy.

Inspired by reddit post and my own thought from the past.

https://www.reddit.com/r/osugame/s/aw5lI4YVYj

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/28138
https://github.com/ppy/osu/discussions/22616
etc
